cmd/build/helmify/main.go

```diff
@@ -10,7 +10,7 @@ import (
 	"regexp"
 	"strings"
 
-	"gopkg.in/yaml.v3"
+	"go.yaml.in/yaml/v3"
```

Comment:
@mrueg should we just move to v4? are there breaking changes from v3 - v4?

Reply:
I wanted to keep the change set minimal. v4 hasn't had an official release. latest is https://github.com/yaml/go-yaml/releases/tag/v4.0.0-rc.3